easybuild/tools/config.py

```diff
@@ -217,6 +217,7 @@ def mk_full_default_path(name, prefix=DEFAULT_PREFIX):
         'cleanup_tmpdir',
         'extended_dry_run_ignore_errors',
         'mpi_tests',
+        'modules_tool_version_check',
```

Comment:
This is conforming naming btw!

I am saying it, because it would allow to configure things on my end, like:
```
EASYBUILD:
  BUILDPATH:                    /dev/shm/$USER
  MODULES_TOOL:                 Lmod
  MODULES_TOOL_VERSION_CHECK:  "Yes" ## If you dare not quote it, it becomes "True" via yaml parser!
  MODULE_SYNTAX:                Tcl
```

ref. https://github.com/Bright-Computing/bic/blob/master/test/etc/profile.definitions/site/EasyBuild.yml
(that gets picked up on node initialisation - working logic, but can be improved)